### ISSUE TITLE
Window frozen after minimize from fullscreen and restore...

### ISSUE THREAD

**Issue by flynx** [nwjs/nw.js#4810]
**To reproduce:**
- run nw binery (bare, no app required),
- in DevTools console:
```
document.body.onclick = function(){ nw.Window.get().minimize() }
nw.Window.get().toggleFullscreen()
```
- restore the window via OS (alt-tab. task-bar, ...)

**Result:**
- the window is frozen and does not respond to user interactions,
- cycling focus from the app window and back fixes the issue.

**Tech. data:**
- nw: v0.14.3
- OS: Windows 10

**Comment by ghostoy** [comment]
This issue should be a regression since Chromium 50. See bug from upstream https://crbug.com/599076.
I'll try to fix it.

**Comment by flynx** [comment]
@ghostoy yep, I found the same report yesterday too...

Thanks!

**Comment by simll** [comment]
My app has encountered this problem does not work properly.
I tried to avoid this problem, and let the nwjs work.
```javascript
let win = nw.Window.get();
let opened = (newWin)=>{
  let restore = false;
  // cancel window xp twice minimize animate
  newWin.x = -9999;
  newWin.y = -9999;
  newWin.width = 0;
  newWin.height= 0;
  newWin.on('close', (event)=>{
    win.show();
    if (!restore) win.close();
    newWin.close(true);
  });
  newWin.on('restore', (event)=>{
    restore = true;
    newWin.close();
  });
  // may be can do something let it better,
  // just like newWin.enterFullscreen(), win.capturePage() for newWin
  newWin.minimize();
  win.hide();
};
// when the bug(#4810) was fixed, just remove that minimize event. 
// 'app/blank.html' is a empty file
win.on('minimize', ()=>nw.Window.open('app/blank.html', {title: document.title}, opened));
```
[demo](https://github.com/simll/nwjs-frozen-patch)

I'm looking forward to it be fixed!

**Comment by ghostoy** [comment]
@simll This issue should be fixed in 0.16.0 based on Chromium M51.

**Comment by ghostoy** [comment]
Issue fixed in upstream at https://crbug.com/613461
